### PR TITLE
feat: (FTL-12541) add implementation to automate the framework, library and redirect uris fetch

### PR DESCRIPTION
**Pull request by pulkitb2**
Closes FTL-12541

### CODE REVIEW

File: src/commands/generate/app.ts [hunk 1]
```diff
@@ -14,10 +14,15 @@ import { vpAdapterService } from '../../services/affinidi/vp-adapter'
 import { createAuth0Resources } from '../../services/generator/auth0'
 import { configureAppEnvironment } from '../../services/generator/env-configurer'
 
+const APPS_INFORMATION_GITHUB_LOCATION = 'samples/auth0-nextjs-nextauthjs/tsconfig.json' // TODO: Change
```
Comment (rbrazhnyk) [inline src/commands/generate/app.ts:17]: Change what?

File: src/helpers/app.ts [hunk 2]
```diff
@@ -1,12 +1,90 @@
+import * as fs from 'fs'
```
Comment (rbrazhnyk) [inline src/helpers/app.ts:1]: Do we need fs?

**Review by rbrazhnyk — APPROVED** [review]

File: src/commands/generate/app.ts (hunk 1 above)
Comment (pulkitb2) [reply src/commands/generate/app.ts]: Ah yes, I had to change the url to the actual file. Had added it as reminder. Pushed a commit with the right one

File: src/helpers/app.ts (hunk 2 above)
Comment (pulkitb2) [reply src/helpers/app.ts]: Removed the unused import

**Review by carlos-affinidi — APPROVED** [review]

**Pull request merged**